Sources/x10/swift_bindings/Device.swift

```diff
@@ -114,8 +114,12 @@ public struct Device {
 
   /// The default `Device`.
   public static var `default`: Device {
+  #if DEFAULT_BACKEND_EAGER
```

Comment:
When would we set `DEFAULT_BACKEND_EAGER`?

Reply:
We should set this flag while compiling TensorFlow in order to get an x10 enabled TensorFlow module.